ezbake/config/public/conf.d/auth.conf

```diff
@@ -0,0 +1,126 @@
+authorization: {
```

Comment:
Confused as to why github is showing this file above as "renamed without changes", but is also showing the contents here?  :sheep: